nltk/test/unit/conftest.py

```diff
@@ -7,9 +7,7 @@ def mock_plot(mocker):
 
     try:
         import matplotlib.pyplot as plt
-
+        mocker.patch.object(plt, 'gca')
```

Comment:
This is the reason why, when you run tests, it opens random python windows for a split-second. Shouldn't do that anymore!